locales/fr.json

```diff
@@ -0,0 +1,95 @@
+{
+  "welcome-back-title": "Bon retour dans Buttercup.",
+  "unlock-archive": "Déverrouillez une archive pour commencer ({os}).",
+  "welcome-title": "Bienvenue dans Buttercup.",
+  "welcome-caption": "Vous n’avez pas encore ajouté d’archive. Pourquoi ne pas en ajouter une ?",
+  "open-archive-file": "Ouvrir un ficier archive",
+  "new-archive-file": "Créer un fichier archive",
```

Comment:
I don't know if that's really important but you translate sometimes 'archive' as 'fichier archive' and sometimes just as 'archive', maybe it would be better to stick to one for consistency.

Reply:
'Ouvrir un fichier archive' is related to 'open-archive-file' as 'Ouvrir une archive' is related to 'open archive'… Do you think it's the same ?
Maybe would it be better to translate 'Ouvrir le fichier de l'archive' + 'Ouvrir une archive'?